settlement/settlement.go

```diff
@@ -94,4 +94,8 @@ type ClientI interface {
 	// GetNextProposer returns the next proposer for this chain in case of a rotation.
 	// If no rotation is in progress, it should return nil.
 	GetNextProposer() (*types.Sequencer, error)
+	// GetRollapp returns the rollapp information.
+	GetRollapp() (*types.Rollapp, error)
+	// GetStateInfo returns the state info for the given index.
+	GetStateInfo(index uint64) (*types.StateInfo, error)
```

Comment:
not ncessary since we only use to get next proposer, and there is already a function for it